Projects/Domains/UserDomain/Interface/Entity/FavoriteSongEntity.swift

```diff
@@ -1,7 +1,8 @@
 import Foundation
 import SongsDomainInterface
 
-public struct FavoriteSongEntity: Equatable {
+/// concurrency를 위한 Model의 Mutable 제거 필요
```

Comment:
```suggestion
// TODO: concurrency를 위한 Model의 Mutable 제거 필요
```
잘보이게 이렇게 주석달아두는건 어떤가요?

Reply:
좋아요